src/util/number/round.js

```diff
@@ -20,13 +21,67 @@ define([
  *   - round( "truncate" )( 123.45 ): 123;
  *   - round( "round" )( 123.45, 0.1 ): 123.5;
  *   - round( "round" )( 123.45, 10 ): 120;
+ *
+ *   Based on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round
+ *   Ref: #376
  */
 return function( method ) {
 	method = method || "round";
 	method = method === "truncate" ? numberTruncate : Math[ method ];
-	return function( value, increment ) {
-		increment = increment || 1;
-		return method( value / increment ) * increment;
+
+	return function( value, incrementOrExp ) {
+		var exp, increment;
+
+		value = +value;
```

Comment:
This conversion is new. Considering the limitations of `isNaN`, should you first check if `value` is `typeof number`, and only then use `isNaN`?